dm/ctl/common/config.go

```diff
@@ -33,6 +38,7 @@ func NewConfig() *Config {
 	fs.BoolVar(&cfg.printVersion, "V", false, "prints version and exit")
 	fs.StringVar(&cfg.ConfigFile, "config", "", "path to config file")
 	fs.StringVar(&cfg.MasterAddr, "master-addr", "", "master API server addr")
+	fs.StringVar(&cfg.RPCTimeoutStr, "rpc-timeout", defaultRPCTimeout, "rpc timeout, default is 10m")
```

Comment:
Since `defaultRPCTimeout`, maybe we should use it in the description string to make sure it's always in sync with the variable.

Reply:
We may use `fs.DurationVar` with `&cfg.RPCTimeout` here.